projects/fishtornado_xr/FishTornado.h

```diff
@@ -51,8 +51,9 @@ class FishTornadoApp
     static FishTornadoApp* GetThisApp();
     float                  GetTime() const { return mTime; }
     float                  GetDt() const { return mDt; }
-    const PerspCamera*     GetCamera() const { return &mCamera; }
-    const Shark*           GetShark() const { return &mShark; }
+
+    const Camera* GetCamera() const;
```

Comment:
not related to your change, but wondering why pointers are returned instead of const ref? :p

Reply:
I'm not sure. That was from the initial version of fish tornado from long ago.